arch/arm/src/lpc54xx/lpc54_ethernet.c

```diff
@@ -251,7 +251,6 @@
  * header.
  */
 
-#define ETHBUF       ((struct eth_hdr_s *)priv->eth_dev.d_buf)
 #define ETH8021QWBUF ((struct eth_8021qhdr_s *)priv->eth_dev.d_buf)
```

Comment:
should we move this to the common place too

Reply:
Done